test/FunctionalTests/Microsoft.OData.Core.Tests/UriParser/Extensions/Binders/ApplyBinderTests.cs

```diff
@@ -150,9 +150,9 @@ public void BindApplyWitGroupByShouldReturnApplyClause()
         [Fact]
         public void BindApplyWitGroupByWithNavigationShouldReturnApplyClause()
         {
-            IEnumerable<QueryToken> tokens = _parser.ParseApply("groupby((MyDog/City))");
+            IEnumerable<QueryToken> tokens = _parser.ParseApply("groupby((MyDog/Color))");

```

Comment:
AFAIR I wanted to use `MetadataBinder` and the path `MyDog/City` is not valid in the metadata. But in the test that I added this I actually used the `MetadataBinder`, instead of `ApplyBinder` with a fake bindMethod. What is the preferred way? So that I can unify them.

Reply:
Got it.